src/BingoGame.Indexer.CA/Processors/BingoGameProcessorBase.cs

```diff
@@ -0,0 +1,51 @@
+using AElf.CSharp.Core;
+using AElfIndexer.Client.Handlers;
+using AElfIndexer.Grains.State.Client;
+using BingoGame.Indexer.CA;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Volo.Abp.ObjectMapping;
+
+namespace BingoGame.Indexer.CA.Processors;
+
+public abstract class BingoGameProcessorBase<TEvent> : AElfLogEventProcessorBase<TEvent,TransactionInfo> where TEvent : IEvent<TEvent>, new()
+{
+    protected readonly ContractInfoOptions ContractInfoOptions;
+    protected readonly IObjectMapper ObjectMapper;
+
+    protected BingoGameProcessorBase(ILogger<BingoGameProcessorBase<TEvent>> logger,
+        IObjectMapper objectMapper, IOptionsSnapshot<ContractInfoOptions> contractInfoOptions) : base(logger)
+    {
+        ObjectMapper = objectMapper;
+        ContractInfoOptions = contractInfoOptions.Value;
+    }
+        protected Dictionary<string, long> GetTransactionFee(Dictionary<string, string> extraProperties)
+    {
+        var feeMap = new Dictionary<string, long>();
+        if (extraProperties.TryGetValue("TransactionFee", out var transactionFee))
+        {
+            feeMap = JsonConvert.DeserializeObject<Dictionary<string, long>>(transactionFee) ??
+                     new Dictionary<string, long>();
+        }
+
+        if (extraProperties.TryGetValue("ResourceFee", out var resourceFee))
+        {
+            var resourceFeeMap = JsonConvert.DeserializeObject<Dictionary<string, long>>(resourceFee) ??
+                                 new Dictionary<string, long>();
+            foreach (var (symbol, fee) in resourceFeeMap)
+            {
+                if (feeMap.TryGetValue(symbol, out _))
```

Comment:
Use `ContainsKey` to check if the dictionary contains the key.

Reply:
noted, changed to ContainsKey